python/mlcroissant/mlcroissant/_src/datasets_test.py

```diff
@@ -147,8 +147,19 @@ def test_nonhermetic_loading(version, dataset_name, record_set_name, num_records
     load_records_and_test_equality(version, dataset_name, record_set_name, num_records)
 
 
+# Non-hermetic test cases (data from the internet) for 1.0 specs version only.
 @pytest.mark.nonhermetic
-def test_load_from_huggingface():
+@pytest.mark.parametrize(
+    ["dataset_name", "record_set_name", "num_records"],
+    [
+        ["bigcode-the-stack/metadata.json", "default", 10]
+    ]
+)
+def test_nonhermetic_loading(dataset_name, record_set_name, num_records):
+    load_records_and_test_equality("1.0", dataset_name, record_set_name, num_records)
+
+@pytest.mark.nonhermetic
+def test_load_from_huggingface_1_0():
```

Comment:
https://datasets-server.huggingface.co/croissant?dataset=mnist&full=true is still in 0.8